### PR TITLE
feature/bash

### PR DESCRIPTION
**Pull request by joelsrzalvarez**

### CODE REVIEW

File: staff/joel-suarez/bash/README.md
Comment (fercostacastro) [inline staff/joel-suarez/bash/README.md:1]: might add some more commands

File: staff/joel-suarez/bash/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# BASH
+Bash is a specific type of shell that is widely used in Unix/Linux systems
+
+# pwd
+Command to see in which rute we are now
+
+Example: ```pwd```
+# touch
+Command to create
```
Comment (PereHDZ) [inline staff/joel-suarez/bash/README.md]: You should specify what is created (a file).
Lo digo pa' tocar los cojones.

File: staff/joel-suarez/bash/README.md
Comment (joelsrzalvarez) [reply staff/joel-suarez/bash/README.md:1]: Hello Fernando, I added some more commands you can check if they are enough now 😉

**Review by joelsrzalvarez — COMMENTED** [review]
Modified.

File: staff/joel-suarez/bash/README.md (hunk 1 above)
Comment (joelsrzalvarez) [reply staff/joel-suarez/bash/README.md]: Hello Pere, you can check now if It's good ty

**Comment by manuelbarzi** [comment]
change to develop base branch

**Comment by joelsrzalvarez** [comment]
> change to develop base branch

Done

**Pull request merged**